src/lib.rs

```diff
@@ -411,7 +410,13 @@ impl<'a> Iterator for IncomingRequests<'a> {
 }
 
 impl Drop for Server {
+    #[allow(unused_must_use)]
     fn drop(&mut self) {
         self.close.store(true, Relaxed);
+        // Connect briefly to ourselves to unblock the accept thread
+        let maybe_stream = TcpStream::connect(self.listening_addr);
+        if let Ok(stream) = maybe_stream {
+            stream.shutdown(Shutdown::Both);
```

Comment:
This can be `let _ = stream.shutdown(Shutdown::Both);`, then we can get rid of `#[allow(unused_must_use)]`